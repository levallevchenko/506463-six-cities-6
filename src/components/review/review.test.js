import React from 'react';
import {render} from '@testing-library/react';
import Review from './review';

it(`Review should render correctly`, () => {

  const review = {
    id: 1,
    user: {userId: 1, avatarUrl: `1`},
    comment: ``,
    date: ``,
    reting: ``
  };

  const {container} = render(
      <Review review={review} />
  );

  expect(container).toMatchSnapshot();
});
