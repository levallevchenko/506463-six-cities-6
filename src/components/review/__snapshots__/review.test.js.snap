// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review should render correctly 1`] = `
<div>
  <li
    class="reviews__item"
  >
    <div
      class="reviews__user user"
    >
      <div
        class="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          class="reviews__avatar user__avatar"
          height="54"
          width="54"
        />
      </div>
      <span
        class="reviews__user-name"
      />
    </div>
    <div
      class="reviews__info"
    >
      <div
        class="reviews__rating rating"
      >
        <div
          class="reviews__stars rating__stars"
        >
          <span />
          <span
            class="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        class="reviews__text"
      />
      <time
        class="reviews__time"
        datetime=""
      >
        Invalid Date
      </time>
    </div>
  </li>
</div>
`;
